src/domain/targeted-messaging/entities/outreach.entity.ts

```diff
@@ -18,4 +18,5 @@ export const OutreachSchema = RowSchema.extend({
   sourceFile: z.string().nullish().default(null),
   sourceFileProcessedDate: z.coerce.date().nullish().default(null),
   sourceFileChecksum: z.string().nullish().default(null),
+  targetAll: z.boolean().default(false),
```

Comment:
Since the default is already defined in the entity, can we remove the default here to ensure a value is explicitly provided?

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-9057e364f06b991e1f8974eec6c1d72658c8120d2a7b7770fbad66cd7965c4c8L18